canonical/ubuntu-advantage-client
# Update apt-json-hook messages

## Proposed Commit Message
Update apt-json-hook messages

Update apt-json-hook messages for new requested format while also non installing the hooks anymore on non-LTS releases.

## Test Steps
Verify that the messages were correctly update (integration test run)
Verify that on non-LTS series, we do not deliver any apt hooks on `/usr/lib/ubuntu-advantage`, but we still have the `20apt-esm-hook.conf` on `/etc/apt/apt.conf.d`

## Desired commit type


## Checklist
